apps/omg_watcher/lib/omg_watcher/exit_processor/in_flight_exit_info.ex

```diff
@@ -80,7 +71,7 @@ defmodule OMG.Watcher.ExitProcessor.InFlightExitInfo do
           oldest_competitor: Utxo.Position.t() | nil,
           eth_height: pos_integer(),
           exit_map: %{
-            non_neg_integer() => %{
+            {:input | :output, non_neg_integer()} => %{
```

Comment:
:exclamation: Idea to change `output index` from integer in `0..7` to tuple: `{:input | :output, integer in 0..4}`